package.json

```diff
@@ -2,72 +2,49 @@
   "name": "visual-config-editor",
   "version": "0.0.0-development",
   "homepage": "https://circleci-public.github.io/visual-config-editor/",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview"
+  },
   "dependencies": {
-    "@circleci/circleci-config-sdk": "v0.9.0-alpha.16",
-    "@craco/craco": "^6.3.0",
-    "@monaco-editor/react": "^4.3.1",
+    "@circleci/circleci-config-parser": "file:../circleci-circleci-config-parser-0.0.0-development.tgz",
```

Comment:
Still using a locally packaged version of the parser. I know the real one isn't published yet, but maybe this should wait for that.